### ISSUE TITLE
Remove compression from server

### ISSUE THREAD

**Issue by jniles** [third-culture-software/bhima#2736]
We've been running BHIMA behind a Nginx proxy in production for quite some time.  We can offload some of the work from NodeJS to nginx and save us the cost of installing a node module and upgrading it.  According to [this SO post](https://stackoverflow.com/q/36928800), there is negligible performance difference (I'm inclined to believe nginx would be a bit faster), so we are okay on the performance front.

Basically, we should replace:
```js
const compress = require('compression');
express.use(compress());
```
with
```nginx
gzip on;
gzip_vary on;
gzip_min_length 10240;
gzip_proxied expired no-cache no-store private auth;
gzip_types text/plain text/css text/xml text/javascript application/x-javascript application/xml;
gzip_disable "MSIE [1-6]\.";
``` 
in the NGINX configuration.
